heron/examples/src/java/com/twitter/heron/examples/CustomGroupingTopology.java

```diff
@@ -18,17 +18,17 @@
 import java.util.List;
 import java.util.Map;
 
-import backtype.storm.Config;
-import backtype.storm.StormSubmitter;
-import backtype.storm.generated.GlobalStreamId;
-import backtype.storm.grouping.CustomStreamGrouping;
-import backtype.storm.task.OutputCollector;
-import backtype.storm.task.TopologyContext;
-import backtype.storm.task.WorkerTopologyContext;
-import backtype.storm.topology.OutputFieldsDeclarer;
-import backtype.storm.topology.TopologyBuilder;
-import backtype.storm.topology.base.BaseRichBolt;
-import backtype.storm.tuple.Tuple;
+import org.apache.storm.Config;
```

Comment:
@billonahill @ttim btw, there is only one Kryo that is being used in this entire codebase: Kryo 3.0.3 (as shown in WORKSPACE file). If our backtype (storm < 1.0) is using Kryo 2.x, then it is confusing to me now.